x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ccr/action/FollowStatsAction.java

```diff
@@ -102,6 +106,13 @@ private static Iterator<ToXContent> innerToXContentChunked(Map<String, Map<Integ
             );
         }
 
+        private static final ToLongFunction<ShardFollowNodeTaskStatus> leaderToFollowerGCPDiff = s -> s.leaderGlobalCheckpoint() - s
```

Comment:
nit: I'd be inclined to inline this, it's only used in the one place.

Reply:
ok